reana_workflow_controller/rest/workflows.py

```diff
@@ -409,6 +411,9 @@ def create_workflow():  # noqa
             logs="",
             git_ref=git_ref,
             git_repo=git_repo,
+            workspace_path=SHARED_VOLUME_PATH
+            + "/"
+            + build_workspace_path(request.args["user"], workflow_uuid),
```

Comment:
We should better change `build_workspace_path()` in `reana-db` to append shared path while creating the workflow